### ISSUE TITLE
Relative import 'six.moves.urllib.parse', should be 'six.moves' (relative-import)

### ISSUE THREAD

**Issue by mgedmin** [PyCQA/pylint#2180]
### Steps to reproduce
1. create a virtualenv
2. install `six` and `pylint`
3. create a `wtf.py` with one line of code:

```python
from six.moves.urllib.parse import urljoin
```
4. run `pylint wtf.py`

### Current behavior

```
************* Module wtf
...
W:  1, 0: Relative import 'six.moves.urllib.parse', should be 'six.moves' (relative-import)
...
```

### Expected behavior

No relative-import error

### pylint --version output

```
pylint 1.9.1, 
astroid 1.6.4
Python 2.7.14 (default, Sep 23 2017, 22:06:14) 
[GCC 7.2.0]
```

**Comment by PCManticore** [comment]
Thanks for the report!

**Comment by rouge8** [comment]
I think this is fixed in the current pylint

**Comment by mgedmin** [comment]
Yup, I cannot reproduce any more with

$ pylint --version
pylint 2.3.1
astroid 2.2.5
Python 3.7.3 (default, Apr  3 2019, 05:39:12) 
[GCC 8.3.0]

**Comment by dominiclondon** [comment]
Still experiencing this issue with pylint 1.9.5.

```
pylint --version
No config file found, using default configuration
pylint 1.9.5,
astroid 1.6.6
Python 2.7.17 (default, May  3 2020, 21:21:37)
[GCC 4.2.1 Compatible Apple LLVM 11.0.0 (clang-1100.0.33.17)]
```

**Comment by Pierre-Sassoulas** [comment]
We're not supporting python 2.7 anymore, and we will not make pylint 1.9 evolves. You need to use python3 to have the latest features.
